### PR TITLE
Proposals/multi tenant

### PR DESCRIPTION
**Pull request by JieXiong9119**
See proposal
Implementation in schema added

### CODE REVIEW

**Review by yan130 — APPROVED** [review]

**Review by macintoshpie — COMMENTED** [review]
Can this be captured by an OccupancyClassification?

**Comment by corymosiman12** [comment]
There are probably many different occupancy classifications that might be multi-tenant (office, mixed commercial, retail, etc.), but even within those it's not always guaranteed. This makes it clear.

**Review by corymosiman12 — APPROVED** [review]

File: proposals/2021/Add MultiTenant.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Implementation
+
+```xml
+      <xs:element name="Buildings" minOccurs="0">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="Building" type="auc:BuildingType" maxOccurs="unbounded">
+              <xs:annotation>
+                <xs:documentation>A building is a single structure wholly or partially enclosed within exterior walls, or within exterior and abutment walls (party walls), and a roof, affording shelter to persons, animals, or property. A building can be two or more units held in the condominium form of ownership that are governed by the same board of managers.</xs:documentation>
```
Comment (nllong) [inline proposals/2021/Add MultiTenant.md:39]: This sounds like a copy/paste from some other document. Should we reference the document here?

Comment (JieXiong9119) [reply proposals/2021/Add MultiTenant.md:39]: > This sounds like a copy/paste from some other document. Should we reference the document here?

It's directly copied from the schema. I'm kind of just quoting the original schema to indicate which element the change is under.

Comment (nllong) [reply proposals/2021/Add MultiTenant.md:39]: Oh, gotcha.

**Review by nllong — APPROVED** [review]

**Pull request merged**